### PR TITLE
Create database

### PR DESCRIPTION
**Pull request by joshualyngdoh**
# Description
- Created a vet-clinic database
- Created an animals table in the database
- Made inserts to the table

# Screenshots

image
image
image
image
image
image
image
image
image

### CODE REVIEW

**Review by CollinsTatang — COMMENTED** [review]
Hi @joshualyngdoh , 

While you made a great effort in this project. Unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have a professional README file.

- [x] Please make sure all main headers from the [README template](https://github.com/microverseinc/readme-template) should be present.
- [x] Please make sure that you add the section **Future Features**, including 1-3 potential features to be added.
- [x] Please remove the Microverse logo from your README file
- [x] Please your commit message should be written in the imperative tense and capitalized. [Read more about how to write descriptive commit messages](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8). Kindly correct that in your next commit massage.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: queries.sql [hunk 1]
```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon%';
+SELECT * FROM animals WHERE date_of_birth>= '2016-01-01' AND date_of_birth<='2019-12-31';
+SELECT * FROM animals WHERE neutered = 'Yes' AND escape_attempts < 3;
+SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
+SELECT name, date_of_birth FROM animals WHERE weight_kg > 10.5;
```
Comment (muneebulrehman) [inline queries.sql]: This query should list the name and escape attempts of animals and not the date of birth. Kindly fix this issue.

File: data.sql [hunk 2]
```diff
@@ -1,5 +1,13 @@
...
+VALUES ('Agumon', '2020-02-3', 'Yes', 10.23, 0);
+
+INSERT INTO animals (name, date_of_birth, neutered, weight_kg, escape_attempts)
+VALUES ('Gabumon', '2018-11-15', 'Yes', 8, 2);
+
+INSERT INTO animals (name, date_of_birth, neutered, weight_kg, escape_attempts)
+VALUES ('Pikachu.', '2021-01-7', 'No', 15.04, 1);
+
+INSERT INTO animals (name, date_of_birth, neutered, weight_kg, escape_attempts)
+VALUES ('Devimon', '2017-05-12', 'Yes', 11, 5);
```
Comment (muneebulrehman) [inline data.sql]: - Optional
Even though SQL accepts **yes** and **no** as boolean values, I would strongly recommend using **true** and **false** as per the community standards.

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @joshualyngdoh , 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Descriptive PR
- Good docs

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShadyShawkat — APPROVED** [review]
## Hi @joshualyngdoh, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**